services/worker/src/worker/job_runners/split/opt_in_out_urls_scan_from_streaming.py

```diff
@@ -240,8 +255,7 @@ def compute_opt_in_out_urls_scan_response(
     if not urls_columns:
         return OptInOutUrlsScanResponse(
             urls_columns=[],
-            opt_in_urls=[],
-            opt_out_urls=[],
+            opt_in_out_urls_source=None,
```

Comment:
maybe something clearer for the users? `urls_file`? `urls_csv_file`? (we already know that we are in the context of opt in and opt out)